cypress/integration/visual-test/Button_spec.js

```diff
@@ -0,0 +1,11 @@
+describe('Button visual regression check', () => {
```

Comment:
Given how similar these tests are, this seems like something we could script (possibly two scripts: one to generate test files for all our existing components, and another that's added to the new component scaffolding script)

Reply:
@dangerismycat agreed - that seems like a good follow up.